nvm.sh

```diff
@@ -2417,6 +2416,7 @@ nvm() {
         local VERSION_PATH
         VERSION_PATH="$(nvm_version_path "${VERSION}")"
         local EXIT_CODE
```

Comment:
this line should be removed, or else the `local` will be run twice, which creates shell output in `zsh`